confgenerator/testdata/valid/linux/metrics-receiver_mongodb/input.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+metrics:
+  receivers:
+    mongodb:
+      type: mongodb
+      collection_interval: 30s
```

Comment:
Remove `collection_interval: 30s` from the base unit test? We can have a `metrics-receiver_mongodb_custom` test if we need to test some custom config, or just move it to `metrics-receiver_mongodb_unix_socket` instead.

For the context, we have an effort coming to consolidate the sample configuration in multiple places, so the following files would become one file:
* `confgenerator/testdata/valid/linux/metrics-receiver_{APPNAME}/input.yaml`
* Sample configurations in the `docs/{APPNAME}.md`
* `integration_test/third_party_apps_data/agent/ops-agent/linux/enable_{APPNAME}`

Reply:
@qingling128 sounds good! 👍  have an upcoming rabbitmq logging one that will resolve it